src/main/java/java/applet/Applet.java

```diff
@@ -46,4 +46,9 @@ public Applet() {
 	public void init() {
 	}
 
+	public String getParameter(String param) {
+		return this.htmlElement.getAttribute((String)("param-").concat(param));
```

Comment:
Hello!
Thanks for the contribution.

I think this cast is not acceptable, probably nicer the following form:
```java
import static jsweet.util.Lang.string;
 ...
string("param-" + param)
```

The other problem is that java's [`.getParameter()`](https://docs.oracle.com/javase/8/docs/api/java/applet/Applet.html#getParameter-java.lang.String-) with the `param` argument is not the same as javascript [.getAttribute()](https://www.w3schools.com/jsref/met_element_getattribute.asp) with  `("param-" + param)`.

I think there'll need to iterate `this.htmlElement`, and return the first child element where `.tagName == "param"`, and it's `.getAttribute("name")` is the needed parameter. After that can return it's `.getAttribute("value")`.

Reply:
Thanks for the comments.  I updated to have a more compatible syntax for params a the original Applet tag as you suggested (instead of the previous "param-xxx" that was intended to be consistent with the data-xxx attributes). As I am not really accustomed to Java there might be another not clean syntax, as the HTMLElement cast, but I have not found other ways to do that.
Thanks for the j4ts-awt-swing candy.

Reply:
Unfortunatly the `getElementsByTagName()` will find all sub-params either (within nested objects), which is not the expected behaviour. You need to iterate over `this.htmlElement.children` HTMLCollection, and check the `.tagName` attribute, because it's iterate over only one depth in the DOM tree.

Reply:
Done